### DISCUSSION TITLE
Request to add Trend Micro vision one observed attack technique events

### DISCUSSION THREAD

**Discussion by dipanjan1823** [IBM/IBM-QRadar-Universal-Cloud-REST-API#252] [Ideas] [answered]
Is it feasible to add

**Comment by ChrisCollinsIBM** [comment]
Hi @dipanjan1823, this is an Open Source repo where customers, partners, and vendors themselves can contribute integrations for QRadar via the Universal Cloud REST API Framework.

I'm sure it's feasible to add support for it via this repo, but ultimately who you're asking is can _anyone_ do it?

Hopefully there are other customers or partners who are looking for the same result and can help fulfil your request, or you have the option to write a workflow yourself and publish it.

**Comment by ChrisCollinsIBM** [comment, marked as answer]
The request has been clarified a bit in https://github.com/IBM/IBM-QRadar-Universal-Cloud-REST-API/issues/247 so closing this discussion.
